packages/contracts/package.json

```diff
@@ -71,6 +72,7 @@
         "ethereumjs-util": "^5.1.1",
         "ethers-contracts": "^2.2.1",
         "lodash": "^4.17.4",
-        "web3": "^0.20.0"
+        "web3": "^0.20.0",
+        "zeppelin-solidity": "^1.8.0"
```

Comment:
So the ERC721 implementation is quite large (split across several contracts) and has additional dependencies within Zeppelin. I was actually relieved to learn there was a node module we could just import, rather than cluttering our own contracts. I could probably derive a barebones ERC721 implementation that satisfies our tests and proxy, although I'm not sure what we would gain. What's the drive behind not wanting this dependency?

Reply:
I'm ok with keeping it.